src/super_gradients/recipes/dataset_params/coco_detection_yolox_dataset_params.yaml

```diff
@@ -27,7 +37,7 @@ train_collate_fn: # collate function for trainset in CoCoDetectionDatasetInterfa
   _target_: super_gradients.training.utils.detection_utils.DetectionCollateFN
 
 cache_dir_path: # path to a directory that will be used for caching (with numpy.memmap).
-cache_train_images: True
-cache_val_images: True
+cache_train_images: False
```

Comment:
Just out of curiosity, is training faster with caching? Why turn it off?

Reply:
Because this requires the user to have enough RAM space, which we cannot assume.

Reply:
Oh, got it, thanks